lib/miq_expression.rb

```diff
@@ -30,6 +30,7 @@ class MiqExpression
     ContainerRoute
     ContainerService
     ContainerTemplate
+    PersistentVolume
```

Comment:
Well actually I believe we want here `PersistentVolume` and not `ContainerVolume` which is the base class and basing reports on it will include other unrelated records (pods volumes).